ADOPTERS.md

```diff
@@ -21,3 +21,5 @@ This is a list of adopters of using PowerShell in production or in their product
 * [PowerShell Universal Dashboard](https://ironmansoftware.com/powershell-universal-dashboard) is a cross-platform web framework for PowerShell. 
   It provides the ability to create robust, interactive websites, REST APIs, and Electron-based desktop apps with PowerShell script. 
   More information about PowerShell Universal Dashboard is available at the [PowerShell Universal Dashboard Docs](https://docs.universaldashboard.io).
+* [System Frontier](https://systemfrontier.com/solutions/powershell/) provides dynamically generated web GUIs and REST APIs for PowerShell and other scripting languages. Enable non-admins like help desk and tier 1 support teams to execute secure web based tools on any platform `without admin rights`. Configure flexible RBAC permissions from an intuitive interface, without a complex learning curve.
+  Script output along with all actions are audited. Manage up to 5,000 nodes for free with the [Community Edition](https://systemfrontier.com/solutions/community-edition/).
```

Comment:
@OneScripter Please start every sentence with new line and fix our static test (spelling).